docs/development/core/server/kibana-plugin-core-server.deprecationsservicesetup.md

```diff
@@ -42,12 +40,12 @@ async function getDeprecations({ esClient, savedObjectsClient }: GetDeprecations
       level: 'warning',
       correctiveActions: {
         manualSteps: [
-            i18n.translate('xpack.foo.deprecations.manualStepOneMessage', {
-              defaultMessage: 'Navigate to the Kibana Dashboard and click "Create dashboard".',
-            }),
-            i18n.translate('xpack.foo.deprecations.manualStepTwoMessage', {
-              defaultMessage: 'Select Foo from the "New Visualization" window.',
-            }),
+           i18n.translate('xpack.foo.deprecations.manualStepOneMessage', {
+             defaultMessage: 'Navigate to the Kibana Dashboard and click "Create dashboard".',
+           }),
+           i18n.translate('xpack.foo.deprecations.manualStepTwoMessage', {
+             defaultMessage: 'Select Foo from the "New Visualization" window.',
+           }),
```

Comment:
Not sure why the `scripts/check_published_api_changes.js` picked this up in this PR.